docs/local.md

```diff
@@ -61,32 +70,51 @@ while [test-gslb2](https://github.com/k8gb-io/k8gb/tree/master/k3d/test-gslb2.ya
 Cluster [edgedns](https://github.com/k8gb-io/k8gb/tree/master/k3d/edge-dns.yaml) runs BIND and acts as EdgeDNS holding Delegated Zone for out test setup and answers
 on port `:1053`.
 
-```shell script
+```sh
 dig @localhost -p 1053 roundrobin.cloud.example.com +short
 ```
-Should return ***two A records*** from both clusters:
+Should return ***two A records*** from both clusters (IP addresses and order may differ):
 ```
-10.43.178.134
-10.43.75.137
+172.20.0.2
+172.20.0.5
+172.20.0.4
+172.20.0.6
+```
+
+You can verify that correct IP addresses of all the nodes in both clusters were populated:
+```sh
+for c in k3d-test-gslb{1,2}; do kubectl get no -ocustom-columns="NAME:.metadata.name,IP:status.addresses[0].address" --context $c; done
```

Comment:
consider something (maybe something a little more modified...) like this:

```Makefile
# Makefile
k3d-test-gslb%:
	kubectl get no -o custom-columns="NAME:.metadata.name,IP:status.addresses[0].address" --context $@;
```
and than....

You can verify that correct IP addresses of all the nodes in both clusters were populated:
```shell
 make k3d-test-gslb1 k3d-test-gslb2
 # kubectl get no -o custom-columns=NAME:.metadata.name,IP:status.addresses[0].address --context k3d-test-gslb1
 # kubectl get no -o custom-columns=NAME:.metadata.name,IP:status.addresses[0].address --context k3d-test-gslb2
```